test/acceptance/features/step_definitions/event-create.js

```
@@ -13,87 +13,92 @@ defineSupportCode(({ Given, Then, When }) => {
 
   Then(/^I verify the event name field is displayed$/, async () => {
     await Events
-      .verify.visible('@eventName')
+      .assert.visible('@eventName')
```

Do we need to hard fail? Would it be more useful to continue with the rest of the tests to get the summary of all errors?

---

thats good point about the summary, but if running on CI it could be expensive to wait for all tests to pass to get a summary.

Fail fast would mean more running of tests but then dealing with one fail at a time, which would mean:
- it fails on CI quick
- then you fix it locally and failing fast locally means dealing with one thing at a time

---

Maybe a symptom of wanting to fail early is because we have so many failures at present... another for the testing discussion